src/application/utils/marina-event.ts

```diff
@@ -7,25 +7,26 @@ export interface MarinaEvent<P extends any> {
   payload: P;
 }
 
-export type NewUtxoMarinaEvent = MarinaEvent<UnblindedOutput>;
-export type SpentUtxoMarinaEvent = MarinaEvent<Outpoint>;
-export type NewTxMarinaEvent = MarinaEvent<TxDisplayInterface>;
+export type NewUtxoMarinaEvent = MarinaEvent<{ utxo: UnblindedOutput; accountID: string }>;
+export type SpentUtxoMarinaEvent = MarinaEvent<{ utxo: Outpoint; accountID: string }>;
```

Comment:
nit: I would replace `Outpoint` by `UnblindedOutput` (we pass the whole utxo so user will be able to access all members)